src/openms/include/OpenMS/FORMAT/FileHandler.h

```diff
@@ -121,7 +122,7 @@ namespace OpenMS
       @exception Exception::ParseError is thrown if an error occurs during parsing
     */
     template <class PeakType>
-    bool loadExperiment(const String& filename, MSExperiment<PeakType>& exp, FileTypes::Type force_type = FileTypes::UNKNOWN, ProgressLogger::LogType log = ProgressLogger::NONE, const bool compute_hash = true)
+    bool loadExperiment(const String& filename, MSExperiment<PeakType>& exp, FileTypes::Type force_type = FileTypes::UNKNOWN, ProgressLogger::LogType log = ProgressLogger::NONE, const bool rewrite_source_file = true, const bool compute_hash = true)
```

Comment:
why didnt you add the new parameter at the end? This will possibly break somebody elses code 

```
loadExperiment(filename, exp, FileTypes::UNKNOWN, ProgressLogger::NONE, true);
```
will now do something very different than before